crates/cheatcodes/src/inspector.rs

```diff
@@ -1102,6 +1114,15 @@ impl Cheatcodes {
         outcome
     }
 
+    pub fn create_with_executor<DB: DatabaseExt>(
```

Comment:
This follows the same pattern as `call_with_executor` to better demarcate the behavior for a custom executor. In general this should be coming from upstream, but they haven't done the same for `CREATE` as for `CALL` probably because it's not needed.